*** Reading local file: /opt/airflow/logs/marketvol1_err_logger/Logger_task/2021-11-24T12:16:06.682152+00:00/1.log
[2021-11-24 12:16:14,793] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: marketvol1_err_logger.Logger_task 2021-11-24T12:16:06.682152+00:00 [queued]>
[2021-11-24 12:16:14,801] {taskinstance.py:851} INFO - Dependencies all met for <TaskInstance: marketvol1_err_logger.Logger_task 2021-11-24T12:16:06.682152+00:00 [queued]>
[2021-11-24 12:16:14,801] {taskinstance.py:1042} INFO - 
--------------------------------------------------------------------------------
[2021-11-24 12:16:14,801] {taskinstance.py:1043} INFO - Starting attempt 1 of 2
[2021-11-24 12:16:14,801] {taskinstance.py:1044} INFO - 
--------------------------------------------------------------------------------
[2021-11-24 12:16:14,817] {taskinstance.py:1063} INFO - Executing <Task(PythonOperator): Logger_task> on 2021-11-24T12:16:06.682152+00:00
[2021-11-24 12:16:14,819] {standard_task_runner.py:52} INFO - Started process 6633 to run task
[2021-11-24 12:16:14,826] {standard_task_runner.py:76} INFO - Running: ['airflow', 'tasks', 'run', 'marketvol1_err_logger', 'Logger_task', '2021-11-24T12:16:06.682152+00:00', '--job-id', '168', '--pool', 'default_pool', '--raw', '--subdir', 'DAGS_FOLDER/log_analyzer.py', '--cfg-path', '/tmp/tmps6gaxf1u', '--error-file', '/tmp/tmp8j69dql_']
[2021-11-24 12:16:14,827] {standard_task_runner.py:77} INFO - Job 168: Subtask Logger_task
[2021-11-24 12:16:14,863] {logging_mixin.py:104} INFO - Running <TaskInstance: marketvol1_err_logger.Logger_task 2021-11-24T12:16:06.682152+00:00 [running]> on host 0faa5da98d7e
[2021-11-24 12:16:14,894] {taskinstance.py:1255} INFO - Exporting the following env vars:
AIRFLOW_CTX_DAG_OWNER=airflow
AIRFLOW_CTX_DAG_ID=marketvol1_err_logger
AIRFLOW_CTX_TASK_ID=Logger_task
AIRFLOW_CTX_EXECUTION_DATE=2021-11-24T12:16:06.682152+00:00
AIRFLOW_CTX_DAG_RUN_ID=manual__2021-11-24T12:16:06.682152+00:00
[2021-11-24 12:16:14,934] {logging_mixin.py:104} INFO - Total number of errors:134
Here are all the errors:
[2021-11-17 05:15:12,306] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-17 05:10:10,151] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-17 05:25:13,439] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-17 05:20:11,073] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:56:57,443] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:50:17,080] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-16 12:58:52,906] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-16 12:53:29,112] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:50:51,696] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:35:38,960] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:50:39,030] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-16 12:59:28,348] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-16 12:54:14,069] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:17:35,212] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:41:40,129] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:12:33,104] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:36:37,920] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-17 05:35:11,579] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-17 05:30:09,288] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:49:04,471] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:51:13,666] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:18:04,425] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:42:20,086] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:13:02,445] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:37:16,481] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:49:53,645] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:17:47,282] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:41:52,640] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:12:43,740] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:36:50,259] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:49:43,643] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-17 05:05:13,215] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-17 05:00:10,460] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:51:38,923] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:49:21,280] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-16 13:00:19,384] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-16 12:55:11,546] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:36:13,030] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-17 05:55:12,478] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-17 05:50:10,803] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:51:24,813] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:17:55,462] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:42:09,870] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:12:53,379] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:37:02,435] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:51:02,102] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-16 12:58:27,221] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-16 12:53:13,135] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:57:21,010] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:50:04,031] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:51:33,493] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:49:09,242] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:49:31,586] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:18:12,983] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:42:26,400] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:13:09,209] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:37:21,440] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-17 05:45:12,556] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-17 05:40:09,801] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-16 13:01:39,225] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-16 12:56:23,031] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:35:48,485] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:57:08,400] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:36:02,192] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:50:27,542] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-16 13:02:44,976] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-16 12:57:00,786] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-17 05:15:14,446] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-17 05:10:11,816] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-17 05:25:15,663] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-17 05:20:12,711] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:56:59,421] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:50:18,912] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-16 12:59:10,973] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-16 12:53:32,960] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:50:53,465] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:35:40,609] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:50:41,605] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-16 12:59:42,659] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-16 12:54:21,868] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:17:38,002] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:41:43,232] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:12:35,402] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:36:40,669] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-17 05:35:13,694] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-17 05:30:10,914] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:49:06,515] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:51:15,854] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:18:08,205] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:42:23,401] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:13:05,937] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:37:18,856] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:49:55,520] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:17:49,712] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:41:55,808] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:12:45,460] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:36:52,159] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:49:45,555] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-17 05:05:15,442] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-17 05:00:12,067] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:51:41,528] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:49:23,103] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-16 13:00:31,237] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-16 12:55:27,544] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:36:14,849] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-17 05:55:14,723] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-17 05:50:12,431] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:51:26,866] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:17:57,943] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:42:12,518] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:12:55,145] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:37:05,594] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:51:03,944] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-16 12:58:41,570] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-16 12:53:17,437] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:57:23,025] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:50:05,907] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:51:35,748] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:49:11,938] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:49:33,997] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:18:14,884] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:42:28,772] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:13:11,032] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 15:37:23,289] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-17 05:45:14,842] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-17 05:40:11,502] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-16 13:02:26,632] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-16 12:56:35,925] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:35:50,471] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:57:10,641] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:36:04,062] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-15 14:50:29,241] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-16 13:02:52,371] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-16 12:57:12,072] {taskinstance.py:1455} ERROR - Bash command failed. The command returned a non-zero exit code.
[2021-11-24 12:16:14,934] {logging_mixin.py:104} INFO - 
[2021-11-24 12:16:14,934] {python.py:118} INFO - Done. Returned value was: None
[2021-11-24 12:16:14,938] {taskinstance.py:1159} INFO - Marking task as SUCCESS. dag_id=marketvol1_err_logger, task_id=Logger_task, execution_date=20211124T121606, start_date=20211124T121614, end_date=20211124T121614
[2021-11-24 12:16:14,963] {taskinstance.py:1220} INFO - 0 downstream tasks scheduled from follow-on schedule check
[2021-11-24 12:16:14,997] {local_task_job.py:146} INFO - Task exited with return code 0
